### PR TITLE
Added the colorscheme with images to showcase it

### PR DESCRIPTION
**Pull request by Reiddragon**

### CODE REVIEW

File: Embark.colorscheme [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+RandomSaturationRange=100
+
+[ForegroundIntense]
+Color=203,227,231
+RandomHueRange=360
+RandomSaturationRange=100
+
+[General]
+Anchor=0.5,0.5
+Blur=false
```
Comment (skbolton) [inline Embark.colorscheme:105]: **Question -** Is defining blur required for a Konsole theme

Ideally the theme is just a palette and doesn't cause any other changes. I think users should be able to set whatever opacity, blur, etc options they want. Is the definition of these options necessary or can it be dropped?

**Review by skbolton — COMMENTED** [review]
I am unfamiliar with Konsole and how it all works. Is adding to the readme on how to install the theme something that could be helpful to others?

File: Embark.colorscheme (hunk 1 above)
Comment (Reiddragon) [reply Embark.colorscheme:105]: they can be safely dropped, but it makes no difference as Konsole would init them on the exact values set there anyway (actually why they're even in the file, created the scheme with Konsole's own editor and it set those values there)

**Comment by Reiddragon** [comment]
> I am unfamiliar with Konsole and how it all works. Is adding to the readme on how to install the theme something that could be helpful to others?

Konsole doesn't have a built in "import theme" button, instead opting to use KDE Discover to download and install themes but that can only install themes from store.kde.org, and so installing themes from other sources (like just yanking theme files from github) usually means manually copying the theme file over to the path specified in the README

**Pull request merged**